test/taskcreator_test.js

```diff
@@ -81,6 +82,51 @@ suite('taskcreator_test.js', function() {
       },
     };
 
+    const defaultHook2 = {
```

Comment:
Could this definition be shortened?  Something like `const defaultHook2 = _.assign(defaultHook, hookId: 'tc-test-hook2')`..